src/corona_hakab_model/agent.py

```diff
@@ -1,9 +1,12 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
 from dataclasses import dataclass
-from generation.connection_types import ConnectionTypes
+from typing import TYPE_CHECKING
```

Comment:
Preferably do not do this,  differences when they are not needed make conflicts and unneded diffs